smap/sources/conf/twitter_conf.ini

```diff
@@ -0,0 +1,14 @@
+[server]
+Port = 8002
+
+[/]
+uuid = 6fae1782-27b2-12e1-2a48-370ecdac5e01
+
+[/twitter]
+type = twitterDriver.TwitterDriver
+Rate = 43200
+consumer_key = 5EvNnwcByDSdb0rxiW5J0mKaT
+consumer_secret = kBU9LfH099KajuZiRJasYJWI6agX6J0CViTY4QciRptLpfoFPX
```

Comment:
Makes sense. Change this to say "get key from https://...." instead?
Then I can go ahead and finish the merge.